### DISCUSSION TITLE
Export to CSV delimiter

### DISCUSSION THREAD

**Discussion by ziggekatten** [apache/superset#17671] [Q&A / Help]
Sometimes our user like to export stuff for various reasons, and doing so causes some headaches.

As standard,  ',' is used as delimiter, but as soon as a file contains text with a ',' it goes tits up as they say.....

In the good old days, doing exports in text format i various apps had options around this. Like selecting delimiter (tab would solve our issue) och encapsulate comma separated stuff with "".

Question: Is there any way of:

- defining delimiter (fex. tab instead of ",")
- defining encapsulation (fex using a " before and after).

Google University does not give any answers.

**Comment by kamalkeshavani-aiinside** [comment]
This is not supported at the moment, you can raise request for this feature if you need,

**Comment by raags** [comment]
Doesn't it work with

```
CSV_EXPORT = {"encoding": "utf-8", "sep": " "}
```

**Reply by ziggekatten** [reply]
I'll give it a try!

**Reply by ziggekatten** [reply]
How did I mis this obvious way of sending params to Pandas!? Now I can get it as i Like. Kudos to @raags !

**Reply by finello76** [reply]
It works!!! Thanks
